### ISSUE TITLE
[USER-v2] Migrate to TokenList to improve handling of legacy bridged USDC on Polygon

### ISSUE THREAD

**Issue by vmichalik** [superfluid-org/superfluid-dashboard#731]
## Overview
**What?**
The legacy bridged token should be overridden to USDC.ex 

**How?**
Migrate the Dashboard to the token list as we previously planned
(Optionally) Drop additional hacky solutions on top to improve the UX

**Why?**

**Deadline:**

**UI/UX Design:**

## Acceptance criteria
 - [ ] ...
 - [ ] Satisfies entire acceptance criteria
 - [ ] Consider communication & visibility
 - [ ] Code review is approved by at least 2 reviewers
 - [ ] Tested by Elvijs/QA & discovered bugs resolved
 - [ ] Documentation updated
 - [ ] Assignee/opened by merges PR
 - [ ] Automated E2E tests for happy path
 - [ ] Code is merged
 - [ ] Observability of critical user path
 - [ ] Steps to access code change as a user

## Links/Resources
 * ...

## Readiness criteria
- [ ] Descriptive title
- [ ] What?
- [ ] Why?
- [ ] How?
- [ ] Acceptance criteria complying with Definition of Done
- [ ] Estimate/Story points
- [ ] Deadline: If not a part of an epic
- [ ] Priority: Reflected as order on the board, walking the board from the top and right
- [ ] Assignee
- [ ] Labels
- [ ] Link to epic: If belonging to an epic
- [ ] A task is always a separate ticket/issue
- [ ] UI/UX Design (optional)
- [ ] Blockers/Prerequisites (optional): Use Zenhub blocked by/blocking feature
